scaffold/src/main/scala/ScaffoldPlugin.scala

```diff
@@ -46,19 +47,23 @@ object ScaffoldPlugin extends sbt.AutoPlugin {
           }
     }
 
-  val scaffoldTask =
-    Def.inputTask {
-      val (name, args) = parser.parsed
-      val folder       = g8ScaffoldTemplatesDirectory.value
-      G8.fromDirectoryRaw(folder / name, baseDirectory.value, args, false)
```

Comment:
If `g8Scaffold` is an input task, would it make sense to parse `--force` in `G8.fromDirectoryRaw` somehow?

Reply:
@eed3si9n Changed to use the `--force` flag. Can you take a look again? Thanks!